### PR TITLE
LPS-196156 Convert old AUI in staging.js

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-196156)

## Changes

Remove not needed file staging.js and the references

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/staging/staging-bar-web/npmscripts.config.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
 module.exports = {
 	build: {
 		bundler: {
-			ignore: ['**/staging.js'],
+			ignore: ['**/*'],
```
Comment (ealonso) [inline modules/apps/staging/staging-bar-web/npmscripts.config.js:9]: @BarbaraCabrera I think we can also delete this file

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by BarbaraCabrera** [comment]
ci:test:stable

**Comment by alessandromalves** [comment]
ci:forward:force

**Comment by alessandromalves** [comment]
The failures are not related to this PR. See https://github.com/brianchandotcom/liferay-portal/pull/142991

**Pull request closed without merging**